bioinformatics_tools/dna_rna_tools/dna_rna_tools.py

```diff
@@ -0,0 +1,77 @@
+def run_dna_rna_tools(*args):
+    *sequence, procedure = args
+
+    if procedure == "transcribe":
+        return apply_to_sequences(sequence, transcribe)
+    elif procedure == "reverse":
+        return apply_to_sequences(sequence, reverse)
+    elif procedure == "complement":
+        return apply_to_sequences(sequence, complement)
+    elif procedure == "reverse_complement":
+        return apply_to_sequences(sequence, reverse_complement)
+    else:
+        return None
+
+
+def valid_rna(sequence):
+    if set(sequence).issubset({"A", "U", "G", "C", "a", "u", "g", "c"}):
+        return True
+    else:
+        return False
+
+
+def valid_dna(sequence):
+  
+    return set(sequence).issubset({'A', 'T', 'G', 'C', 'a', 't', 'g', 'c'})
```

Comment:
Интересное решение проблемы с распознаванием ДНК или РНК! Но почему нельзя было сразу обе проверки нуклеиновых кислот сделать как valid_dna(), где сразу будет возвращаться True или False?

И на мой взгляд вместо 2 функций можно было сделать 1 (см. код)
```suggestion
def valid_nucl(sequence):
    return set(sequence).issubset({'A', 'T', 'G', 'C', 'U', 'a', 't', 'g', 'c', 'u'})
```
Тогда при проверке, если у нас нормальные последовательности РНК или ДНК, функция будет выдавать False, а при проблемах - True